lib/sidekiq/monitor/stats.rb

```diff
@@ -28,6 +28,24 @@ def process_metrics
           }
         end
       end
+
+      def job_metrics
+        jobs = []
+
+        Sidekiq::Workers.new.each do |process, thread, msg|
```

Comment:
`Sidekiq::Workers#each` is overridden. So, I didn't use `map` .
https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/api.rb#L919

Reply:
I referenced  this source.
https://github.com/mperham/sidekiq/blob/master/web/views/busy.erb#L81L99

Reply:
Regarding `Sidekiq::Workers#each` being overridden... since [the class does `include Enumerable`](https://github.com/mperham/sidekiq/blob/4deadba37200bf7812109a2d6de2371727b2ee3f/lib/sidekiq/api.rb#L917) it should be fine to use `map` here. I just did a quick try and it seems to work.

Reply:
Oh, it was my mistake! Thank you! 😄

Reply:
Fixed https://github.com/harvesthq/sidekiq-monitor-stats/pull/4/commits/bd8f2471915a8fda113e945b620f04b6f1e610ca